### ISSUE TITLE
'Depsgraph' object has no attribute 'update'

### ISSUE THREAD

**Issue by jedStevens** [godotengine/godot-blender-exporter#229]
**OS:**
Manjaro

On the `blender-2.8` branch 

**Blender version:**
Hash: 141c6073ca39

**Issue description:**
When trying to export a scene I get this error:

```
Traceback (most recent call last):
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/__init__.py", line 198, in execute
    return export_godot.save(self, context, **keywords)
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/export_godot.py", line 318, in save
    exp.export()
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/export_godot.py", line 267, in export
    self.export_scene()
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/export_godot.py", line 211, in export_scene
    self.export_object(obj, root_gd_node)
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/export_godot.py", line 153, in export_object
    self.export_object(child, exported_node)
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/export_godot.py", line 123, in export_object
    parent_gd_node)
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/converters/mesh.py", line 45, in export_mesh_node
    mesh_id = mesh_exporter.export_mesh(escn_file, export_settings)
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/converters/mesh.py", line 250, in export_mesh
    mesh = mesh_converter.to_mesh()
  File "/home/jed/.config/blender/2.80/scripts/addons/io_scene_godot/converters/mesh.py", line 706, in to_mesh
    depsgraph.update()
AttributeError: 'Depsgraph' object has no attribute 'update'

location: <unknown location>:-1
```

I originally thought this was due to my scene coming from Blender 2.79 originally however even trying to export the default cube scene causes this error. I assume this means something is up with my current system, but since GLTF export is causing more errors for me once inside godot, I would like to improve this exporter instead.

It just sucks not having a working exporter is all, as I'm at a stage of trying to get animations out of blender and if I could create the animations in Godot I would do that instead. Its just odd that I don't see this error in the issues already.

Any ideas?

**Comment by Jason0214** [comment]
Looks like you are using a relatively old build. The python API of blender2.8 has rapidly changed through last few months. Maybe give a try about a recent blender2.8 build?

Or you can use the [build](https://travis-ci.org/godotengine/godot-blender-exporter/builds/552484956#L290) from CI, this one should work with the current HEAD

**Comment by jedStevens** [comment]
It seems that the blender version coming from the `yaourt` package manager was causing conflicts with the `blender-2.8` exporter branch. I'm getting other odd errors now but seemingly unrelated to this.

**Comment by Jason0214** [comment]
Looks like your build is from Feb, you can reset the exporter to this [commit](https://github.com/godotengine/godot-blender-exporter/commit/3c1df895819455e310e6caaf63f3660b09bdd730), hopefully it would work

**Comment by Calinou** [comment]
Closing, as this should be resolved now that Blender 2.80 has a stable release.
